src/cobot_pump_node.cpp

```diff
@@ -17,12 +17,14 @@
 #include "cobot_pump_ros/startPump.h"
 #include "cobot_pump_ros/stopPump.h"
 #include "cobot_pump_ros/dropItem.h"
+#include "cobot_pump_ros/readState.h"
+#include "cobot_pump_ros/checkItemAttached.h"
 
 using namespace std;
 
 // TODO - make this ip changeable when you create the node, i.e via arguments
 // Instantiate vacuum gripper with our ip address
-// franka::VacuumGripper vacuum_gripper("192.168.130.1");
+franka::VacuumGripper vacuum_gripper("192.168.130.1");
```

Comment:
We can pass the IP to the node as an argument (for example, see [here](https://answers.ros.org/question/299014/command-line-argument-passing-in-rosrun/?answer=299124#post-id-299124)). We can pass a `_franka_ip` or something like that. This will be annoying for someone running the node from command line (we need to pass IP every time), but if a roslaunch file is created, we can hard-code the IP in the ROS launch file instead of the C++ file.